content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md

```diff
@@ -388,6 +392,8 @@ TARGET_VERSION=2.2.1
 STORAGE_MEDIA_TYPE=application/json
 ```
 
+** Note: this upgrade procedure is for getting from 2.x to 3.x.  Version `3.0.17` is not recommended for running in production (see [prerequisites](#prereqisites) for minimum recommended etcd versions.
```

Comment:
Looks great!

Reply:
Why single out 3.0.17?  The guidance I had been seeing previously said just get it above 3.0.10.  https://groups.google.com/forum/#!topic/etcd-dev/nZQl17RjxHQ dumps on all of 3.0 --- and more emphatically than I am seeing here.  Somewhere there should be maintained an evergreen doc for general etcd use like https://groups.google.com/forum/#!topic/etcd-dev/nZQl17RjxHQ

Reply:
Just saying `3.0` here sounds fine.

@MikeSpreitzer For the evergreen doc, that would be for the etcd docs? We should probably move discussion to github.com/etcd-io/etcd.

Reply:
Yes, @jpbetz , I think the etcd docs should include a maintained statement like what you sent out in email.

Reply:
I don't want to change the tested upgrade procedure.
I updated the advice to not  single out 3.0.17, and just suggest further upgrading to the recommended version of etcd.